<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../res/icons/classification.png</normaloff>../res/icons/classification.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockLabel">
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>80</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Labels</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="dockEffects">
   <property name="minimumSize">
    <size>
     <width>82</width>
     <height>150</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Effects</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QDockWidget" name="dockFeatures">
   <property name="minimumSize">
    <size>
     <width>180</width>
     <height>80</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Features</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open Cloud Point File</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
   <property name="statusTip">
    <string>Save 3D Model</string>
   </property>
  </action>
  <action name="actionDataAnalysisSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Data analysis settings</string>
   </property>
   <property name="statusTip">
    <string>Data analysis settings</string>
   </property>
  </action>
  <action name="actionDataAnalysis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Data analysis</string>
   </property>
  </action>
  <action name="actionNumClasses">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set number of classes</string>
   </property>
   <property name="toolTip">
    <string>Set number of classes</string>
   </property>
   <property name="statusTip">
    <string>Set number of classes</string>
   </property>
  </action>
  <action name="actionCompute_features">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Compute features</string>
   </property>
   <property name="iconText">
    <string>Compute features</string>
   </property>
   <property name="toolTip">
    <string>Compute features</string>
   </property>
   <property name="statusTip">
    <string>Introduce el numero de caracteriticas geometricas a utilizar.</string>
   </property>
  </action>
  <action name="actionWeightFeatures">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Weights settings</string>
   </property>
   <property name="statusTip">
    <string>Select the geometric characteristics</string>
   </property>
  </action>
  <action name="actionEffectFeatures">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Effects settings</string>
   </property>
   <property name="statusTip">
    <string>Select effect of each geometric characteristic</string>
   </property>
  </action>
  <action name="actionClassification">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Classification</string>
   </property>
  </action>
  <action name="actionNumFeatures">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set number of features</string>
   </property>
   <property name="statusTip">
    <string>Set number of features</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="statusTip">
    <string>Close all documents</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionType_of_classification">
   <property name="text">
    <string>Type of classification</string>
   </property>
   <property name="statusTip">
    <string>Select the type of classificaton</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
